packages/cashier/src/pages/deposit/__tests__/deposit.spec.tsx

```diff
@@ -85,7 +84,7 @@ describe('<Deposit />', () => {
         };
 
         const { rerender } = render(<Deposit setSideNotes={jest.fn()} />, {
-            wrapper: ({ children }) => <StoreProvider store={mockRootStore}>{children}</StoreProvider>,
+            wrapper: ({ children }) => <StoreProvider store={mockRootStore as TRootStore}>{children}</StoreProvider>,
```

Comment:
It is okay to use `as` in the test files as we use `DeepPartial` to provide only the part of the data that is needed for the test case.